floor_generator/lib/writer/query_method_writer.dart

```diff
@@ -126,7 +126,10 @@ class QueryMethodWriter implements Writer {
     @nullable final String arguments,
     @nonNull final String mapper,
   ) {
-    final parameters = StringBuffer()..write("'${_queryMethod.query}', ");
+    final parameters = StringBuffer()
+      ..write('r""" ${_queryMethod.query} """')
```

Comment:
❓ Did you add the leading and trailing space on purpose?
```suggestion
      ..write('r"""${_queryMethod.query}"""')
```

Reply:
See above